src/humanReadablePane.js

```diff
@@ -86,6 +81,28 @@ module.exports = {
       console.log('humanReadablePane: unknown content-type?')
     }
 
+    //  @@ When we can, use CSP to turn off scripts within the iframe
+    div.setAttribute('class', 'docView')
+    const iframe = myDocument.createElement('IFRAME')
+    let dataUri
+
+    // render markdown to html
+    const markdownHtml = function () {
+      kb.fetcher.webOperation('GET', subject.uri).then(response => {
+        const res = marked.parse(response.responseText)
+        dataUri = 'data:text/html;charset=utf-8,' + encodeURIComponent(res)
+        iframe.setAttribute('src', dataUri) // allow-same-origin
```

Comment:
**suggestion (non-blocking)** you could just render the markdown html into a div, no need for an iframe in this case

Reply:
I decided to keep the same html display for both. It is a lazy solution. The performance is quite good.

Reply:
I am fine with that